docs/modules/ROOT/pages/network-files.adoc

```diff
@@ -0,0 +1,64 @@
+= Network Files
+
+OpenZeppelin Upgrades keep track of all the implementation contracts you have deployed in an `.openzeppelin` folder in the project root, as well as the proxy admin. You will find one file per network there. It is advised that you commit to source control the files for all networks except the development ones (you may see them as `.openzeppelin/unknown-*.json`).
+
+> Note: the format of the files within the `.openzeppelin` folder is not compatible with those of the xref:cli::index.adoc[OpenZeppelin CLI]. If you want to use these plugins for an existing OpenZeppelin CLI project, we will be sharing soon a guide on how to migrate.
+
+
+[[network.json]]
+== `<network_name>.json`
+
+OpenZeppelin Upgrades will generate a file for each of the networks you work on (`ropsten`, `mainnet`, etc). These files share the same structure:
+
+[source,json]
+----
+// .openzeppelin/<network_name>.json
+{
+  "manifestVersion": "3.0",
+  "impls": {
+    "...": {
+      "address": "...",
+      "txHash": "...",
+      "layout": {
+        "storage": [...],
+        "types": {...}
+      }
+    },
+    "...": {
+      "address": "...",
+      "txHash": "...",
+      "layout": {
+        "storage": [...],
+        "types": {...}
+      }
+    }
+  },
+  "admin": {
+    "address": "...",
+    "txHash": "..."
+  }
+}
+----
+
+For every logic contract, besides the deployment address, the following info is also tracked:
+
+* `types` keeps track of all the types used in the contract or its ancestors, from basic types like `uint256` to custom `struct` types
+* `storage` tracks the storage layout of the linearized contract, referencing the types defined in the `types` section, and is used for verifying that any storage layout changes between subsequent versions are compatible
```

Comment:
Here we could have a link to the FAQ entry about storage compatibility.

Reply:
Added link: https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/142/commits/c00466b4ce08e97f8149af712e1d409165297958